DotaStats/AppCoordinator.swift

```diff
@@ -58,6 +58,17 @@ extension AppCoordinator {
             imageNetworkService: StubImageNetworkService()
         )
     }
+
+    private func matchInfoModuleBuilder() -> MatchInfoModuleBuilder {
```

Comment:
Фабричные методы лучше всего начинать со слова `make`
```suggestion
    private func makeMatchInfoModuleBuilder() -> MatchInfoModuleBuilder {
```